src/main/java/edu/unc/lib/boxc/migration/cdm/services/ListProjectsService.java

```diff
@@ -90,11 +101,9 @@ public JsonNode listProjects(Path directory) throws Exception {
     private String status(MigrationProject project) {
         String status = null;
 
-//        TODO: add archived state
-//        if () {
-//            status = "archived";
-//        }
-        if (!project.getProjectProperties().getSipsSubmitted().isEmpty()) {
+        if (project.getProjectPath().toString().toLowerCase().contains(ArchiveProjectsService.ARCHIVED)) {
```

Comment:
While its unlikely to happen, I think we should check for `/.archived/` to appear in the string, to make sure we don't match something like `/path/to/gilmer.archived` or similar. 

It also doesn't need to be lowercased, since paths are case sensitive, so if it were `.ARCHIVED` we wouldn't list any projects in the directory anyways.